features/membership-application-status/admin-custom-reason-waiting.feature

```diff
@@ -17,7 +17,7 @@ Feature: "Other/Custom" waiting reason comes from locale file and Admin cannot e
   Background:
 
     # it is important that this statement is first so that tables are empty, so that things will be seeded
-    Given the system is seeded with initial data
+    # Given the system is seeded with initial data
```

Comment:
Since we now add the "custom reason" selection directly (no longer from the DB) this background step is not needed.